### ISSUE TITLE
Set stationNumber for competing assignments in WCIF file

### ISSUE THREAD

**Issue by DallasMcNeil** [jonatanklosko/groupifier#55]
Hello,

I'm working on a system to generate name badges and schedules for Australian competitions. I'm using the WCIF file for competitions for all the information. We used fixed seating so station numbers for competitors are very important and traditionally we haven't been able to include them on name badges, requiring competitors to search for their station number on a sheet or at dropoff. I'm looking to include station numbers on personal schedules and I've noticed the WCIF file includes a field for 'stationNumber' but it isn't populated. 

`wcif.persons[x].assignments[y].stationNumber == null`

Is this something groupifier can do? This would be extremely useful and although I'm not familiar with the system, I can't imagine it would be very difficult since this information is already generated. I'd even be happy to develop this myself if you point me in the right direction. 

Regards
Dallas

**Comment by KerrieJarman** [comment]
+1 to this.  This would be a game changer for our competitions.  It is really hard work to have hundreds of newcomers at a competition and trying to convey where they need to drop off their puzzle and to sit.  We NEED station numbers to be on the individual schedules and to live in the WCIF (ie not just generated on the fly with scorecards)

**Comment by jonatanklosko** [comment]
I made a change to add station numbers to WCIF when generating/changing assignments (provided that "Print out station number" is enabled). Those are then used when generating scorecards and should match the on-the-fly numbers we generated so far. I hope this helps with generating the custom schedules, if there's any inconsistency let me know :)

The user still needs to make sure groups are small enough, such that everybody has a station available. So there still isn't proper support for fixed-sitting, but that is more involved, because it would require checks in a number of places.
